### DISCUSSION TITLE
Control tab is not showing anything

### DISCUSSION THREAD

**Discussion by seng1218** [vladmandic/sdnext#3376] [General]
Hi to all, it may sound stupid but i exhausted all the wiki and i couldn't find the answer. 
Why my control tab in sd next is not showing anything? Is there anything I've done wrong or missing steps? 
Hope to get reply. Thanks in advance. 

![Screenshot 2024-08-21 090217](https://github.com/user-attachments/assets/f2835653-920f-4f7e-80b9-c4ffc699e98a)

**Comment by vladmandic** [comment]
Control tab is available when using backend:diffusers (default).
If you switch to backend:original, it's not available (many options are not available in original backend).

**Comment by seng1218** [comment]
Thank you so much, really appreciate.
